app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,31 @@
 
   <body>
     <nav class = "topnav">
-      <%= link_to "All Merchants", "/merchants"%>
-      <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Register", "/register"%>
+      <%= link_to "Home", "/" %>
+      <%= link_to "All Merchants", "/merchants" %>
+      <%= link_to "All Items", "/items" %>
+      <% if current_user && current_user.regular? %>
```

Comment:
There's opportunity for code re-use here if 'regular_user_logged_in?' etc is available in this scope as a helper method in 'ApplicationController'